mbs_results/outputs/selective_editing_question_output.py

```diff
@@ -92,19 +86,13 @@ def create_selective_editing_question_output(
         sic_mapping="sic_5_digit",
     )
 
-    df_with_domain = calculate_predicted_value(
```

Comment:
This is no longer needed? From code i can tell that this is passed as a dimension but is this relevant to ASAP-494?

Reply:
The calculate predicted value function just combines them into a single column in a different way using np.where instead of summing. From what I understood it was a fix to combine then, but this is called after the temp fix on 480 and once you summed imputed and target, it would always return the same column if that makes sense. I will double check this function isnt needed

Reply:
Have double checked with Luke, the `calculate_predicted_value` function is not needed as this is another method of combining adjusted and imputed values together before producing outputs. Relates to 494 as this is a downstream adjustement which can be removed if we combine adjusted and imputed values earlier in pipeline